simpleocr/files.py

```diff
@@ -28,6 +29,18 @@ def try_extensions(path, extensions):
     return None
 
 
+def find_file(file_name):
```

Comment:
I agree with you that this needs to be fixed, but this is unrelated to this PR - can you please submit it afterwards? It's harder to review code that keeps changing
When there's a bug in the current codebase, It's a good practice to create a test that fails before fixing the bug - to make sure it's actually broken (repeatedly), and that the fix actually fixes it :)